TopQuarkAnalysis/TopEventProducers/interface/TopDecaySubset.h

```diff
@@ -43,12 +47,34 @@ class TopDecaySubset : public edm::EDProducer {
  private:
   /// find top quarks in list of input particles
   std::vector<const reco::GenParticle*> findTops(const reco::GenParticleCollection& parts);
+  /// find primal top quarks (top quarks from the hard interaction)
+  /// for Pythia6 this is identical to findDecayingTops
+  std::vector<const reco::GenParticle*> findPrimalTops(const reco::GenParticleCollection& parts);
```

Comment:
Hi Luke,

I am not sure about the unit test for this package.
But you already communicated with TOP PAG conveners and agreed each other,
and this does not affect old functionality,
so I don't see any problem to have this necessary new features for Run 2. 

Thank you for the development.

Regards,
Taejeong